assets/js/map.js

```diff
@@ -0,0 +1,203 @@
+(function(exports) {
+
+  var map = d3.select("#team-map")
+        .classed("loading", true),
+      status = map.append("p")
+        .html("Loading the map...");
+
+  // names and geographic locations of airport codes
+  var locations = [
+    {code: "DCA", label: "Washington", location: [-77.037722, 38.852083]},
+    {code: "SFO", label: "San Francisco", location: [-122.374889, 37.618972]},
+    {code: "CHI", label: "Chicago", location: [-87.631667, 41.883611]},
+    {code: "DAY", label: "Dayton", location: [-84.219375, 39.902375]},
+    {code: "DEN", label: "Denver", location: [-104.673178, 39.861656]},
+    {code: "PHL", label: "Philadelphia", location: [-75.241139, 39.871944]},
+    {code: "NYC", label: "New York", location: [-74.005833, 40.714167]},
+    {code: "TUS", label: "Tuscon", location: [-110.941028, 32.116083]},
+    {code: "SEA", label: "Seattle", location: [-122.309306, 47.449]},
+    {code: "AUS", label: "Austin", location: [-97.669889, 30.194528]}
+  ];
+
+  var urls = {
+    team: "/api/team/",
+    topology: "/assets/data/us-states.json"
+  };
+
+  queue()
+    .defer(d3.json, urls.team)
+    .defer(d3.json, urls.topology)
+    .await(function(error, team, topology) {
```

Comment:
Also you might want to name that async function to ensure you get better traces on error.